### DISCUSSION TITLE
Label centet alignment

### DISCUSSION THREAD

**Discussion by annunal** [beeware/toga#2362] [Debugging help]
When I use Label I think that thus should put the text in the center of the label:

Lab=toga.Label("my text", style=Pack(width=300, alignment='center'))

But in reality is is always left aligned.

May I get help or indications in relation to ?

Thanks

**Comment by rmartin16** [comment]
Try using [`text_align`](https://toga.readthedocs.io/en/stable/reference/style/pack.html#text-align) for the text of a widget; [`alignment`](https://toga.readthedocs.io/en/stable/reference/style/pack.html#alignment) focuses on the children of a widget.

**Reply by annunal** [reply]
Thanks a lot, it works.
